src/main/kotlin/no/nav/familie/ef/sak/vedtak/PatchPeriodetypeBarnetilsynController.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.familie.ef.sak.vedtak
+
+import no.nav.familie.ef.sak.vedtak.domain.BarnetilsynWrapper
+import no.nav.familie.ef.sak.vedtak.domain.Periodetype
+import no.nav.familie.ef.sak.vedtak.dto.ResultatType
+import no.nav.security.token.support.core.api.Unprotected
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.http.MediaType
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/api/periodetype-barnetilsyn-patch"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@Unprotected
+@Validated
```

Comment:
@Validated kan sikkert slettes?

Reply:
Har denne noe å si? 
Føler ikke den har noe å si for denne controllern, og heller ikke PR-messig dessverre.

Eller tenker du at vi burde fjerne den på generell basis?